wiki/Beatmaps/en.md

```diff
@@ -1,3 +1,28 @@
+### Beatmaps
```

Comment:
this was a quick snippet dump so I don't forget to put it here, I will not fix it in this pr